libs/payments/capability/src/lib/capability.types.ts

```diff
@@ -0,0 +1,5 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export type ClientIdCapabilityResult = Record<never, string[]>;
```

Comment:
Record<never, string[]> doesn't exactly make much sense, since it's saying that this is an object in the following format:

```
{
  never: ["asdf"]
}
```

In other words, no keys for the object _can_ exist, so you're basically saying ClientIdCapabilityResult can only ever be a value of `{}`, which doesn't make much sense.

Reply:
Removed.